### ISSUE TITLE
npm install failed Failed at the @sentry/cli@1.71.0 install script.

### ISSUE THREAD

**Issue by octsama** [getsentry/sentry-cli#1089]
### Environment

OS:CentOS 7.0
Node:v13.12.0
npm:6.14.13

### Steps to Reproduce

1. npm install

### Expected Result

npm install succeed

### Actual Result

```
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @sentry/cli@1.71.0 install: `node scripts/install.js`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the @sentry/cli@1.71.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-12-22T15_51_37_255Z-debug.log
```

[2021-12-22T15_51_37_255Z-debug.log](https://github.com/getsentry/sentry-cli/files/7763780/2021-12-22T15_51_37_255Z-debug.log)

**Comment by harshcut** [comment]
**Environment**

- OS: Windows 11
- Node: v14.18.2
- NPM: v6.14.15

**Actual Result**

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @sentry/cli@1.71.0 install: `node scripts/install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @sentry/cli@1.71.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\harsh\AppData\Roaming\npm-cache\_logs\2021-12-29T04_11_21_177Z-debug.log
```

**Comment by Awais6** [comment]
same error here

**Comment by raviten** [comment]
any workarounds?

**Comment by Awais6** [comment]
> any workarounds?

I have downloaded the sentry-cli exe file for windows and opened cmd at the same path and pass all parameters to that file. It will work the same. If you want to run sentry-cli from the whole computer define that exe file path in environmental variables in the path directory.

**Comment by kamilogorek** [comment]
This issue is caused by incorrect permissions for filesystem operations. Whether on your local machine, VM, or CI setup. See https://github.com/getsentry/sentry-cli/issues/281 for some possible solutions.

tl;dr is that install script was not able to successfully store downloaded binary, which then failed to be spawned.

**Comment by kyawzinhtett** [comment]
use VPN
